roles/grafana-docker/defaults/main.yml

```diff
@@ -4,14 +4,13 @@ grafana_config_path: "/etc/grafana"
 grafana_data_path: "/var/lib/grafana"
 grafana_docker_image: "grafana/grafana:{{ grafana_version }}"
 grafana_console_port: "3000"
-grafana_url: "http://localhost:3000"
+grafana_url: "http://{{ ansible_ssh_host }}:3000"
 grafana_admin_user: "admin"
 grafana_admin_pwd: "secret"
 
-grafana_influxdb_url: "http://localhost:8086"
+grafana_influxdb_url: "http://{{ ansible_ssh_host }}:8086"
```

Comment:
The same comment as with `roles/riemann-docker/defaults/main.yml:riemann_influxdb_host.`